### PR TITLE
Meeting help dialog

### PR DESCRIPTION
**Pull request by ackernaut**
Addresses #1835 

- Uses `<LoadableMenu />` to create a portal with toggle
- Uses a look-up file to provide content depending on the phase
  - Content is for the help dialog and facilitator control bar
  - We may not always use a tip for the facilitator control bar based on context, other controls present
- Needs:
  - [x] Real content, including for both Retro and Action lobbies
  - To be wired into the `<MeetingControlBar />` of real meeting views (TBD in a new issue)

Take a gander using `yarn storybook`

### CODE REVIEW

**Comment by ackernaut** [comment]
This is ready and generally supports the following IA (with flexibility when to use UI affordances) and I’ve got [a doc of meeting UI copy](https://docs.google.com/document/d/1daME__aPxBPuPNQw7lAVyeREVPkheNdWV2sdl1NclbM/edit) for the internal team.

meeting ia

**Comment by ackernaut** [comment]
@mattkrick can you review this when you get a chance? Thank you.

File: src/universal/utils/meetings/helpLookups.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+);
+
+const teamAgendaHelpLink = makeLink('https://www.parabol.co/getting-started-guide#team-agenda', 'Learn More');
+
+const actionGettingStartedLink = makeLink('https://www.parabol.co/getting-started-guide', 'Getting Started Guide');
+// TODO: Add Learn More links, etc. for Retro Guide when it is created
+const retroGettingStartedLink = makeLink('https://www.parabol.co/getting-started-guide', 'Getting Started Guide');
+
+// TODO: Refactor to handle Retro type
+const meetingLabel = 'an Action Meeting' || 'a Retro Meeting';
```
Comment (mattkrick) [inline src/universal/utils/meetings/helpLookups.js]: check out lookups.js, see `meetingTypeToLabel`

File: stories/Portals.stories.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <StoryContainer
+        description={`This demonstrates a help dialog (available for all teammates)
+          and an optional tip (for the facilitator in the control bar)`}
+        render={() => (
+          <div style={{padding: '50vh 0 0'}}>
+            <div style={{display: 'flex', justifyContent: 'flex-end', padding: '0 1.25rem 1.25rem 0'}}>
+              <MeetingHelpDialog phase={EXAMPLE_PHASE} />
+            </div>
+            <MeetingControlBar>
+              {phaseHelpLookup[EXAMPLE_PHASE].facilitatorBarTip}
```
Comment (mattkrick) [inline stories/Portals.stories.js]: my only concern here is when we show these.
A few of these should probably only be triggered on an action, eg the voting phase requirements. 
then again, i think the conditional requirements is out of scope of this PR, so i think it's good to start here!

Comment (ackernaut) [reply stories/Portals.stories.js]: Agreed! When we start using this I think I may just hide some of them until we can make them conditional. I’ll consider as the criteria for an issue to wire up.

File: src/universal/utils/meetings/helpLookups.js (hunk 1 above)
Comment (ackernaut) [reply src/universal/utils/meetings/helpLookups.js]: Will do!

**Pull request merged**